src/main/java/org/spongepowered/api/service/context/Context.java

```diff
@@ -31,11 +31,12 @@
 import java.util.Map;
 
 /**
- * The context that a permission check occurs in. Instances of a context are
+ * The context that a service check occurs in. Instances of a context are
  * designed to function as cache keys, meaning they should be fairly lightweight
  * and not hold references to large objects
  */
 public final class Context implements Map.Entry<String, String> {
+    public static final String USER_KEY = "user";
```

Comment:
Would this allow Accounts to be shared by players, or only be fixed to a single player?